### ISSUE TITLE
Use multiline text for text button

### ISSUE THREAD

**Issue by daybyter** [ImpulseAdventure/GUIslice-Builder#147]
Hi!

Just tried to use a text like "line1\nline2" for a button, but it seems the guislice builder doesn't wrap the text as I would expect it?

Is there some feature, that has to be checked for this feature to be enabled?

Thanks in advance,
Andreas

**Comment by Pconti31** [comment]
No. Its just not currently supported by the Builder.  The multiple lines do work with Adafruit's GFX but not with all drivers that GUIslice API supports.  TFT_eSPI for example just eats the newline.
I'll consider this for a future release.

**Comment by MikeyMoMo** [comment]
I need this now for a project.  I hope it can be added soon.

**Comment by Pconti31** [comment]
@MikeyMoMo Checked in support Release 0.17.b03. Tested using Arduino mega and Adafruit GFX libraries.
NOTE: Sizing of a Text button using multiple lines of text is the users responsibility. You will need to set 
the height and width manually.  Ending each line inside the Label with \n. Example, "line1\nline2\nline3".
The Builder will remove each \n and replace with a decimal 10 (newline) when you press enter key.

I attached a zip file the sample I created for testing.
[ex_multiline.zip](https://github.com/ImpulseAdventure/GUIslice-Builder/files/7961359/ex_multiline.zip)

Paul--
